targets/ar71xx-generic

```diff
@@ -251,18 +251,50 @@ if (env.GLUON_REGION or '') ~= '' then
 	tplink_region_suffix = '-' .. env.GLUON_REGION
 end
 
-device('tp-link-cpe210-v1.0', 'cpe210-220-v1', {
-	aliases = {'tp-link-cpe210-v1.1', 'tp-link-cpe220-v1.1'},
+device('tp-link-cpe210-v1', 'cpe210-220-v1', {
+	aliases = {
+		'tp-link-cpe220-v1',
```

Comment:
either i don't understand it or here is an issue, please check again

Reply:
The device will be listed as CPE210 v1 and CPE220 v1 in the sysupgrade / factory output directories (with the latter being a symlink / alias). These are the names that users will look for when searching for images for their devices.